Oracle/PCORNetLoader_ora.sql

```diff
@@ -48,85 +48,86 @@ BEGIN
   EXECUTE IMMEDIATE sqlstring;
   dbms_output.put_line(sqlstring);
 END PMN_ExecuateSQL;
-/
 
 
 
 
 
 
-CREATE OR REPLACE SYNONYM I2B2FACT FOR I2B2DEMODATA.OBSERVATION_FACT
-/
+
+CREATE OR REPLACE SYNONYM I2B2FACT FOR I2B2DEMODATA.OBSERVATION_FACT;
 
 
 BEGIN
 PMN_DROPSQL('DROP TABLE i2b2patient_list');
 END;
-/
+
 
 CREATE table i2b2patient_list as 
 select * from
 (
 select DISTINCT PATIENT_NUM from I2B2FACT where START_DATE > to_date('01-Jan-2010','dd-mon-rrrr')
 ) where ROWNUM<100000000
-/

```

Comment:
I'd rather not take this change, just because I can't fully test Oracle stuff here - we've been relying on one of our sites to do that. Can you remove it from the pull request?